.travis.yml

```diff
@@ -35,10 +35,12 @@ cache:
 
 jobs:
   include:
-    - script: COVERAGE=1 npm start test.node
+    - before_script: node scripts/test-runner-filter.js || travis_terminate 0
```

Comment:
can we not use `before_install`?

Reply:
Then the node modules are not in place to run the script. We could rewrite it to a bash script I guess

Reply:
that's probably reasonable, but doesn't need to happen in this PR.